modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -163,6 +163,10 @@ def form_is210966
         params[:form_number] == '21-0966'
       end
 
+      def form_is210966_and_should_use_itf_api
+        form_is210966 && loa3 && icn && first_party?
+      end
```

Comment:
A better name for this method might be `use_itf_api_for_210966_form?`

Reply:
Agreed and fixed! Thank you!